hibernate-core/src/test/java/org/hibernate/orm/test/jpa/metamodel/MixedIdAndIdClassHandling.java

```diff
@@ -50,8 +54,16 @@ public void testAccess(EntityManagerFactoryScope scope) {
 		assertEquals( 1, entityType.getSupertype().getIdClassAttributes().size() );
 
 		assertFalse( entityType.hasSingleIdAttribute() );
+	}
 
-		assertEquals( String.class, entityType.getIdType().getJavaType() );
```

Comment:
Why did you delete this assertion? Was that an accident?

Reply:
No, `entityType.getIdType().getJavaType()` returns `EmployeeId.class` now, this behavior is covered by another dedicated test.

Reply:
Sorry, `entityType.getIdType().getJavaType()` still returns `String.class` for back compatibility, but the assertion is moved to dedicated  test `testGetIdType()`.

Reply:
I see now.